{
    "topic": "TS Task Errors",
    "pageNumber": "1",
    "type": "TS Task Error - Unexpected Error",
    "definition": {
        "text": "Write here a summary for this topic page."
    },
    "paragraphs": [
        {
            "style": "Text",
            "text": "Left click and Edit to enter edit mode and change this text. ENTER to write new paragraphs. ESC to exit edit mode."
        },
        {
            "style": "Error",
            "text": "Unexpected Error"
        },
        {
            "style": "Title",
            "text": "Error Message",
            "updated": 1610971316249
        },
        {
            "style": "Text",
            "text": "The following is the Error Message of the Exception that occured at the TS:",
            "updated": 1610971326601
        },
        {
            "style": "Placeholder",
            "text": "errorMessage",
            "updated": 1610971337737
        },
        {
            "style": "Title",
            "text": "Error Stack",
            "updated": 1610968600444
        },
        {
            "style": "Text",
            "text": "The following is the Error Stack of the Exception that occured at the TS:",
            "updated": 1610968631281
        },
        {
            "style": "Placeholder",
            "text": "errorStack",
            "updated": 1610968640796
        },
        {
            "style": "Title",
            "text": "Error Code"
        },
        {
            "style": "Text",
            "text": "The follwing is the Error Code of the Exception that occured at the TS:"
        },
        {
            "style": "Placeholder",
            "text": "errorCode"
        },
        {
            "style": "Text",
            "text": ""
        }
    ]
}